src/app/cube/home/learning-object-info/learning-outcomes/learning-outcomes.component.scss

```diff
@@ -1,15 +1,77 @@
-.content{
+@import 'vars.scss';
+
+.content {
     margin-top: 50px;
     margin-right: 50px;
+    padding-left: 0px;
     display: flex;
     flex-direction: column;
-    .taxonomy{
-      margin-bottom: 400px;
+    
+    .blooms-cards {
+        display: flex;
+        flex-direction: row;
```

Comment:
Adding a flex-wrap will make this more responsive. Without it, it loses responsiveness between about 1295px width and regains it at about 944px.